databricks/koalas/utils.py

```diff
@@ -455,31 +455,58 @@ def validate_bool_kwarg(value, arg_name):
     return value
 
 
-def verify_temp_column_name(sdf: spark.DataFrame, column_name: str) -> str:
+def verify_temp_column_name(df: Union["ks.DataFrame", spark.DataFrame], column_name: str) -> str:
     """
```

Comment:
Can we also add a short description what's `temp_column_name`? For example, we can say ...

The temporary column names should start and end with `__`. In addition, `column_name` only expects single string instead of labels when `df` is a Koalas DataFrame.